imsim/instcat.py

```diff
@@ -399,67 +400,65 @@ def getHourAngle(self, mjd, ra):
         return ha
 
 class OpsimMetaDict(object):
-    """This just handles the meta information at the start of the instance catalog file.
+    """"Read the exposure information from the opsim db file.
 
     The objects are handled by InstCatalog.
     """
-    _req_params = { 'file_name' : str, }
+    _req_params = { 'file_name' : str,
+                    'visit' : int,
+                    'snap' : int}
```

Comment:
Maybe make snap optional, with default 1?  (Or 0?)  Not sure how often users will actually care to distinguish the snap number.

Reply:
The current cadences have visits with either 1 or 2 exposures per visit, as well as different exposure times depending on the band.   Putting the snap here is a near term way to address this.  Ultimately, I would think we'd want the code to generate as many exposures as specified for the requested visit, but I wasn't sure what changes that imply for the current code, so I thought we could address that in a separate PR.